pkg/user/helper.go

```diff
@@ -0,0 +1,26 @@
+package user
+
+import (
+	"github.com/Layr-Labs/eigenlayer-cli/pkg/internal/common"
+	"github.com/Layr-Labs/eigensdk-go/logging"
+	gethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/urfave/cli/v2"
+)
+
+func PopulateCallerAddress(
```

Comment:
we could technically put this function in a more generic way where allocations commands can also use it. Instead of account address we can just say defaultAddress  so we can pass account or operator address.

Reply:
we can push that in separate PR. should be fine for now

Reply:
Talked offline -- agree and thought about this but opting for easier solution to keep flags specific to their use-cases for now and will address in the CLI refactor workstream.